docs/serverless-v2/06-01-function-cr.md

```diff
@@ -52,12 +75,40 @@ This table lists all the possible properties of a given resource together with t
 | **spec.deps** | No | Specifies the lambda's dependencies. |
-| **spec.function** | Yes | Provides the lambda's source code. |
+| **spec.minReplicas** | No | Defines the minimum number of lambda's Pods to run at a time. |
+| **spec.maxReplicas** | No | Defines the maximum number of lambda's Pods to run at a time. |
+| **spec.resources.limits.cpu** | No | Maximum number of CPUs available for the lambda's Pod to use. |
+| **spec.resources.limits.memory** | No | Maximum amount of memory available for the lambda's Pod to use. |
+| **spec.resources.requests.cpu** | No |  Number of CPUs requested by the lambda's Pod to operate. |
+| **spec.resources.requests.memory** | No | Amount of memory requested by the lambda's Pod to operate. |
+| **spec.source** | Yes | Provides the lambda's source code. |
+| **status.conditions.lastTransitionTime** | Not applicable | Provides a timestamp for the last time the lambda's condition status changed from one to another. |
+| **status.conditions.message** | Not applicable | Describes a human-readable message on the CR processing progress, success, or failure.  |
+| **status.conditions.reason** | Not applicable | Provides information on the Function CR processing success or failure. See the [**Reasons**](#status-reasons) section for the full list of possible status reasons and their descriptions. All status reasons are in camelCase. |
+| **status.conditions.status** | Not applicable | Describes the status of processing the Function CR by the Function Controller. It can be `True` for success, `False` for failure, or `Unknown` if the CR processing is still in progress. If the status of all conditions is `True`, the overall status of the Function CR is ready. |
+| **status.conditions.type** | Not applicable | Describes a substage of the Function CR processing. There are three condition types that a lambda has to meet to be ready: `ConfigurationReady`, `BuildReady`, and `Running`. When displaying the lambda status in the terminal, these types are shown under `CONFIGURED`, `BUILT`, and `RUNNING` columns respectively. All condition types can change asynchronously depending on a type of lambda modification, but all three need to be in the `True` status for the lambda to be considered successfully processed. |
+
+### Status reasons
+
+Processing of a Function CR can succeed, continue, or fail for one of these reasons:
+
+| Reason | Type | Description |
+| --------- | ------------- | ----------- |----------- |
+| `ConfigMapCreated` | `ConfigurationReady` | A new ConfigMap was created based on the Function CR definition. |
+| `ConfigMapUpdated` | `ConfigurationReady` | The existing ConfigMap was updated after changes in the Function CR name, its source code or dependencies. |
+| `ConfigMapError` | `ConfigurationReady` | The ConfigMap couldn't be created or updated due to an error. |
+| `JobFailed` | `BuildReady` | The image with the lambda's configuration couldn't be created due to an error. |
```

Comment:
I'm not changing it, because it's not an error and also because maybe I missed something :D However, a question: have we changed the approach to contractions: https://kyma-project.io/community/guidelines/content/#style-and-terminology-style-and-terminology-contractions ?
Refers also to line 109.

Reply:
I believe we loosened up this approach on one of the content meetings but that is not a problem - I can remove them if that may cause confusion.